examples/phyast.py

```diff
@@ -0,0 +1,390 @@
+import phylanx
+et = phylanx.execution_tree
+import inspect
+import ast
+import re
+
+def phy_print(m):
```

Comment:
This function is duplicated in three files, should it be separated?

Reply:
It should only exist once now.